extensions/control-plane/api/management-api/transfer-process-api/src/main/java/org/eclipse/edc/connector/api/management/transferprocess/transform/JsonObjectToTransferRequestTransformer.java

```diff
@@ -54,7 +60,12 @@ public JsonObjectToTransferRequestTransformer() {
 
         builder.id(nodeId(input));
         visitProperties(input, k -> switch (k) {
-            case TRANSFER_REQUEST_CONNECTOR_ADDRESS -> v -> builder.connectorAddress(transformString(v, context));
+            case TRANSFER_REQUEST_CONNECTOR_ADDRESS -> v -> {
+                monitor.warning(format("The attribute %s has been deprecated in type %s, please use %s",
+                        TRANSFER_REQUEST_CONNECTOR_ADDRESS, TRANSFER_REQUEST_TYPE, TRANSFER_REQUEST_COUNTER_PARTY_ADDRESS));
```

Comment:
Since working a `Monitor` into the existing `Validator` interface would be a big change in multiple places not related to this PR, I reckon this should either be a seperate Issue or a new `ValidatorWithMonitor` Interface should be created specificly for this case. Any thoughts here?

Reply:
My suggestion would be to pass the monitor in the constructor, like we usually do with dependency injection.

Reply:
Moved the warning messages from transformers to their respective validators.